### PR TITLE
journal-week-1 (branch) 

### PR DESCRIPTION
**Pull request by sjoyal**

### CODE REVIEW

**Comment by MStaehling** [comment]
:+1:

File: TIY-week-1-survival-story.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+**Coding Bootcamp: Tips for surviving week 1**
+  * 1) Only stop working on classwork and/or additional JavaScript practice to eat, sleep, shower, and drive
+    * Think I'm kidding? This blog was assigned for class. Get off me.
+  * 2) Vary the topic / assignment you're working on with regularity
+    * Learning JavaScript defnitions is necessary. It's also not fun. Do some fun online codeschool programs to reinforce the definitions and/or any other assignments that need to be done. Vary the routine. It helps me.
+  * 3) It's never too early to build something
+    * See earlier comment about online codeschools not meaning a whole lot when you're staring at a blank .js file
+    * Be reasonable with your expectations of what you can accomplish starting out
+    * I built a paper rock scissors game; It's lame, it took me 6 hours, and it just runs on the command line, but I'm 72% sure I built it right...which is nice.
+  * 4) Drink hot tea because everyone else drinks coffee, and that stuff stunts your growth.
```
Comment (gatorpazz) [inline TIY-week-1-survival-story.md]: How much bigger do you think you're going to grow junior?

**Comment by gatorpazz** [comment]
TL;DR You learned lots of stuff and are excited to learn more yippeee, +1

**Comment by al-the-x** [comment]
This is great feedback @MStaehling @gatorpazz... `</sarcasm>`
